web-components/src/components/menu-overlay/MenuOverlay.stories.ts

```diff
@@ -83,6 +83,7 @@ export const MenuOverlay = () => {
           </div>
         </div>
       </md-menu-overlay>
+      <iframe style="float: left; margin-top: 200px;" title="iframe-to-test-menu-closing" src="http://127.0.0.1:8080/?path=/story/components-date-range-picker--date-range-picker"></iframe>
```

Comment:
Thought I would leave this sample iframe in the example to ensure this behaviour stays working as any other updates are made